credentials/src/main/kotlin/web5/sdk/credentials/PresentationExchange.kt

```diff
@@ -21,19 +21,17 @@ public object PresentationExchange {
   /**
    * Selects credentials that satisfy a given presentation definition.
    *
-   * @param credentials The list of Verifiable Credentials to select from.
+   * @param vcJwts Iterable of VCs in JWT format to select from.
    * @param presentationDefinition The Presentation Definition to match against.
    * @return A list of Verifiable Credentials that satisfy the Presentation Definition.
-   * @throws UnsupportedOperationException If the method is untested and not recommended for use.
    */
   @Throws(UnsupportedOperationException::class)
   public fun selectCredentials(
-    credentials: List<VerifiableCredential>,
+    vcJwts: Iterable<String>,
     presentationDefinition: PresentationDefinitionV2
-  ): List<VerifiableCredential> {
-    throw UnsupportedOperationException("pex is untested")
-    // Uncomment the following line to filter credentials based on the Presentation Definition
-    // return credentials.filter { satisfiesPresentationDefinition(it, presentationDefinition) }
+  ): List<String> {
```

Comment:
An alternative API is one that returns the indices of the credentials which where selected. Specifically, see below

```kt
public fun selectCredentials(
    vcJwts: Iterable<String>,
    presentationDefinition: PresentationDefinitionV2
  ): List<Int> {
```

Returning indexes allows the consumers of the library to more easily choose what to do with the result. For instance, if the consumer has to show greyed out VCs within their wallet like application, using indices seems like it would be much easier than having to compare the full strings of VCs. 

Curious what your thoughts are?

Reply:
So following web5-js and the pex library that we use underneath it they return a list of credentials. To your point the library also returns a SelectResults object which has a lot more things in it like warnings, errors, and indexes along side the actual credential jwts.

I think there could be discussion for a large Results object for many of these PEX methods in the future, but for right now I think it makes the most sense to just return a list of the actual selected credential JWTs.

also an iterable could be a set or map that doesn't guarantee that the index will line up